flow360/component/simulation/framework/entity_registry.py

```diff
@@ -27,6 +27,10 @@ def entities(self):
         """Return all entities in the bucket."""
         return self._source.get(self._key, [])
 
+    def get_list_of_specified_property(self, property_name: str):
+        """Get the given property for all the entities in the bucket"""
+        return [getattr(entity, property_name) for entity in self.entities]
```

Comment:
not sure if I understand intention of this function. It seems to be not used anywhere except tests. We can make it private? Also, maybe `get_property_values` is better name?

Reply:
Renamed to `_get_property_values`. This might be used in future help functions. But then still should be private.